report-viewer/src/components/DistributionDiagram.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="wrapper">
+    <BarChart class="chart" :chartData="chartData" :options="options"/>
+  </div>
+</template>
+
+<script>
+import {defineComponent, ref, watch} from "vue";
+import { BarChart } from "vue-chart-3"
+import { Chart, registerables } from 'chart.js';
+import ChartDataLabels from 'chartjs-plugin-datalabels';
+Chart.register(...registerables);
+Chart.register(ChartDataLabels);
+
+export default defineComponent({
+  name: "DistributionDiagram",
+  components: { BarChart },
+  props: {
+    distribution: {
+      type: Array,
+      required: true
+    }
+  },
+  setup(props) {
+    //Highest count of submissions in a percentage range. We set the diagrams maximum shown value to maxVal + 5,
+    //otherwise maximum is set to the highest count of submissions and is one bar always reaches the end.
+    let maxVal = ref(Math.max(...props.distribution));
+
+    let chartData = ref( {
+        labels: ['0-10%', '11-20%', '21-30%', '31-40%', '41-50%', '51-60%', '61-70%', '71-80%', '81-90%', '91-100%'],
+        datasets: [{
+          label: 'Count',
+          data: props.distribution,
+          backgroundColor: 'rgba(149, 168, 241, 0.5)',
```

Comment:
Can this be derived from the main color definitions?

Reply:
I could not access the CSS variables from here.